transform_to_tf/src/library/transform_to_tf.cpp

```diff
@@ -0,0 +1,51 @@
+#include "transform_to_tf/transform_to_tf.hpp"
+
+namespace transform_to_tf {
+
+TransformToTf::TransformToTf(const ros::NodeHandle& nh,
+                             const ros::NodeHandle& nh_private)
+    : nh_(nh),
+      nh_private_(nh_private),
+      verbose_(kDefaultVerbose),
+      local_frame_name_(kDefaultLocalFrameName),
+      global_frame_name_(kDefaultGlobalFrameName) {
+  // Getting data and params
+  subscribeToTopics();
+  advertiseTopics();
+  getParametersFromRos();
+}
+
+void TransformToTf::subscribeToTopics() {
+  // Subscribing to the required data topics
+  transform_sub_ = nh_.subscribe("transform_to_be_converted", 1,
+                                 &TransformToTf::transformCallback, this);
+}
+
+void TransformToTf::advertiseTopics() {
+  // Advertising topics
```

Comment:
Not sure advertise is the best description of what is happening in this function